pkg/plugins/manager/manifest.go

```diff
@@ -205,7 +206,37 @@ func getPluginSignatureState(log log.Logger, plugin *plugins.PluginBase) (plugin
 		var unsignedFiles []string
 		for _, f := range plugin.Files {
 			if _, exists := manifestFiles[f]; !exists {
-				unsignedFiles = append(unsignedFiles, f)
+				fullPath, err := filepath.Abs(filepath.Join(plugin.PluginDir, f))
+				if err != nil || !filepath.IsAbs(fullPath) {
+					log.Error("Absolute file path is required to provide plugin symlink verification", "file", f)
+					return plugins.PluginSignatureState{}, err
```

Comment:
err would be nil here if `!filepath.IsAbs(fullPath)` is true. Is that expected?

Reply:
No I should separate these actually 😄